cypress/plugins/index.js

```diff
@@ -51,15 +51,21 @@ module.exports = (on, config) => {
 
   const waitUntilAppRestarts = async (timeout) => await waitOn({ delay: 2000, resources: [config.baseUrl], timeout })
   const getReplacementText = async (text, source) => source ? fsp.readFile(source) : text
-  const replaceText = ({ text, originalText, newText, source }) => getReplacementText(newText, source)
-    .then((replacementText) => {
-      return text.replace(originalText, replacementText)
-    })
+  const replaceText = ({ text, originalText, newText, source }) => {
+    return getReplacementText(newText, source)
+      .then((replacementText) => {
+        if (text.includes(originalText)) {
+          return text.replace(originalText, replacementText)
+        } else {
+          throw new Error('Text to be replaced not found')
+        }
+      })
+  }
 
   const replaceMultipleText = async (text, list) => {
     let resultText = text
     let index = 0
-    while (list.length >= index) {
+    while (index < list.length) {
```

Comment:
What is this change about?

Reply:
I was getting a failure because it was comparing against an entry in the list outside the list of text to replace.

I otherwords the list may have 5 entries and the index was running right up to 5 but the maximum can only be 4.